src/actions.ml

```diff
@@ -30,21 +35,34 @@ let send_status_check ~bot_info job_info pr_num (gh_owner, gh_repo)
         >>= function
         | Ok {issue= id; head; last_commit_message= Some commit_message}
         (* Commits reported back by get_pull_request_refs are surrounded in double quotes *)
-          when String.equal head.sha (f "\"%s\"" job_info.commit) ->
-            GitHub_mutations.post_comment ~bot_info ~id
-              ~message:
-                (f
-                   "For your complete information, the job \
-                    [%s](https://gitlab.com/%s/-/jobs/%d) in allow failure \
-                    mode has failed for commit %s: %s%s"
-                   job_info.build_name repo_full_name job_info.build_id
-                   job_info.commit
-                   (first_line_of_string commit_message)
-                   ( if
-                     String.equal job_info.build_name
-                       "library:ci-fiat_crypto_legacy"
-                   then "\nping @JasonGross"
-                   else "" ))
+          when String.equal head.sha (f "\"%s\"" job_info.commit) -> (
+            let message =
+              f
+                "For your complete information, the job [%s](%s) in allow \
+                 failure mode has failed for commit %s: %s%s"
+                job_info.build_name job_url job_info.commit
+                (first_line_of_string commit_message)
```

Comment:
Since this is going to be displayed in the Checks tab, the message should be adjusted. In particular, there is no need to adopt a discussion-like language ("For your complete information") nor is there to specify for which commit the build has failed.